resources/measures/HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1245,177 +1211,23 @@ def self.process_infiltration_ventilation(runner, mj8, unit, thermal_zone, zone_
-            # Convert to 24 hour values
-            sched_values = []
-            previous_time_decimal = 0
-            day_sched.times.each_with_index do |time, i|
-              time_decimal = (time.days * 24.0) + time.hours + (time.minutes / 60.0) + (time.seconds / 3600.0)
-              # Back-fill in hourly values
-              for hr in sched_values.size + 1..time_decimal.floor
-                sched_values[hr - 1] = day_sched.values[i]
-              end
-            end
-          elsif sched.is_a? OpenStudio::Model::ScheduleFile
-            runner.registerError("ScheduleFile schedule type for '#{sched.name}' is not supported.")
-            return nil
-          else
-            runner.registerError("Schedule type for '#{sched.name}' is not supported.")
-            return nil
-          end
-        end
-        if not max_mult.nil?
-          # Calculate daily load
-          annual_energy = Schedule.annual_equivalent_full_load_hrs(@year_description, sched) * design_level_w * gain.multiplier # Wh
-          daily_load = UnitConversions.convert(annual_energy, "Wh", "Btu") / Constants.NumDaysInYear(@year_description.isLeapYear) # Btu/day
-          # Calculate design level in Btu/hr
-          design_level = sched_values.max * daily_load * max_mult # Btu/hr
-          # Normalize schedule values to be max=1 from sum=1
-          sched_values_max = sched_values.max
-          sched_values = sched_values.collect { |n| n / sched_values_max }
-        end
-      elsif sched.is_a? OpenStudio::Model::ScheduleConstant
-        sched_values = [sched.value] * 24
-      else
-        runner.registerError("Unexpected type for object '#{sched.name.to_s}' in process_internal_gains.")
-        return nil
-      end
-      if sched_values.size != 24
-        runner.registerError("Expected 24 schedule values for object '#{gain.name.to_s}' but found #{sched_values.size} values.")
-        return nil
-      end
-
-      for hr in 0..23
-        int_Sens_Hr[hr] += sched_values[hr] * design_level * sensible_frac
-        int_Lat_Hr[hr] += sched_values[hr] * design_level * latent_frac
-      end
-    end
-
-    # Process occupants
-    n_occupants = nbeds + 1 # Number of occupants based on Section 22-3
-    occ_sched = [1.00000, 1.00000, 1.00000, 1.00000, 1.00000, 1.00000, 1.00000, 0.88310, 0.40861, 0.24189, 0.24189, 0.24189,
-                 0.24189, 0.24189, 0.24189, 0.24189, 0.29498, 0.55310, 0.89693, 0.89693, 0.89693, 1.00000, 1.00000, 1.00000]
     zone_ffa = Geometry.get_finished_floor_area_from_spaces(thermal_zone.spaces)
-    for hr in 0..23
-      int_Sens_Hr[hr] += occ_sched[hr] * 230 * n_occupants * zone_ffa / unit_ffa
-      int_Lat_Hr[hr] += occ_sched[hr] * 200 * n_occupants * zone_ffa / unit_ffa
-    end
 
-    return int_Sens_Hr, int_Lat_Hr
+    # Per ANSI/RESNET/ICC 301
+    n_occupants = nbeds + 1
+    zone_loads.Cool_IntGains_Sens = 1600.0 + 230.0 * n_occupants * zone_ffa / unit_ffa
```

Comment:
This is missing parentheses, needs to be: (1600.0 + 230.0 * n_occupants) * zone_ffa / unit_ffa